cogent/INSTALL.md

```diff
@@ -88,3 +90,15 @@ maintain a Stack build scheme.
   $> cabal test
 ```
 
+
+### Testing on macOS
+The default `cpp` bundled with macOS isn't compatible with Cogent – running `make examples` will fail.
```

Comment:
Apart from `make examples`, there are a lot of other things that won't work without an appropriate`CC`. It might be useful to note what you CAN do without it.